gammapy/data/event_list.py

```diff
@@ -50,20 +54,58 @@ class EventList(Table):
     - `radec` for ``RA``, ``DEC``
     - `energy` for ``ENERGY``
     - `galactic` for ``GLON``, ``GLAT``
+
+    Examples
+    --------
+
+    To load an example H.E.S.S. event list:
+
+    >>> from gammapy.data import EventList
+    >>> filename = '$GAMMAPY_EXTRA/test_datasets/unbundled/hess/run_0023037_hard_eventlist.fits.gz'
+    >>> events = EventList.read(filename)
+
+    To load an example Fermi-LAT event list (the one corresponding to the 2FHL catalog dataset):
+
+    >>> filename = '$GAMMAPY_EXTRA/datasets/fermi_2fhl/2fhl_events.fits.gz'
+    >>> events = EventList.read(filename)
     """
 
-    def summary(self, file=None):
+    def __init__(self, table):
+        self.table = table
+
+    @classmethod
+    def read(cls, filename, **kwargs):
+        """Read :ref:`gadf:iact-events`
+
+        Parameters
+        ----------
+        filename : `~gammapy.extern.pathlib.Path`, str
+            Filename
+        """
+        filename = make_path(filename)
+        if 'hdu' not in kwargs:
+            kwargs.update(hdu='EVENTS')
+
+        table = Table.read(str(filename), **kwargs)
+        return cls(table=table)
+
+    def info(self, file=None):
```

Comment:
Please change this to ``__str__`` (and add an info method calling it if you like). see http://docs.gammapy.org/en/latest/development/howto.html#object-summary-info-string